cloudinit/config/cc_ssh.py

```diff
@@ -165,11 +172,15 @@
 
 CONFIG_KEY_TO_FILE = {}
 PRIV_TO_PUB = {}
+PRIV_TO_CERT = {}
```

Comment:
On re-review, I noticed this `PRIV_TO_CERT` isn't used anywhere outside of here and the block below. I believe we can remove it.